blog/zh/blog/2023/10/09/wecity-uses-apisix.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "荷兰智慧城市革命：WeCity 与 APISIX 的合作之旅"
+authors:
+  - name: Arjen Hof, Tim van Densen
+    title: Author
+    url:
+    image_url:
+  - name: Jing Yan
+    title: Technical Writer
+    url:
+    image_url: https://avatars.githubusercontent.com/u/114121331?v=4
+keywords:
+  - Case Studies
+description: 荷兰知名公共数据管理企业 WeCity 与 APISIX 携手同行，共同打造荷兰的稳定安全的数据市场。
+tags: [Case Studies]
+image: https://static.apiseven.com/uploads/2023/09/20/6ZpgFEMr_WeCity.png
+---
+
+> “开源项目 APISIX 的运营理念开放灵活，自身功能强大齐全，社区响应沟通及时，优点相当瞩目。”
+> 
+> ——WeCity 联合创始人兼首席技术官Arjen Hof、WeCity 软件架构师兼首席开发人员 Tim van Densen
+
+作者：WeCity 联合创始人兼首席技术官Arjen Hof、WeCity 软件架构师兼首席开发人员 Tim van Densen
+
+编译：API7.ai 增长实习生 颜菁
+
```

Comment:
```suggestion
```

Reply:
You should add yourself to authors part as you put efforts in translation. 

```
  - name: 颜菁
	    title: translator
	    url: https://github.com/JYan00
	    image_url: https://github.com/JYan00.png
```